<ui version="4.0" >
 <class>SampleItem</class>
 <widget class="QWidget" name="SampleItem" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="m_lblQuestionNumber" >
     <property name="text" >
      <string>Question number</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="m_lblQuestionText" >
     <property name="text" >
      <string>Question text</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="m_lblAnswerText" >
     <property name="text" >
      <string>Answer text</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>265</width>
       <height>71</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" >
    <widget class="QLabel" name="m_lblWarning" >
     <property name="text" >
      <string/>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="m_btnNext" >
       <property name="text" >
        <string>&amp;Next</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_btnPrevious" >
       <property name="text" >
        <string>&amp;Submit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
